tools/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/crc-org/vfkit/tools
 
-go 1.21
+go 1.22.1
```

Comment:
@cfergeau once we take this in and then update `vfkit` for crc then on crc side also go 1.22.x will be required.

Reply:
It's in `vfkit/tools`, not in `vfkit`, `crc` does not use `vfkit/tools` so this should not force a 1.22.x update in crc.
I tried with this in crc: `go mod edit --replace github.com/crc-org/vfkit=github.com/cfergeau/vfkit@lint` followed by `go mod tidy`, and go.mod is still using `go 1.21`.